Bottom: bda5812ec90fc053c47bdc013b9c23c17b3ea89f
Top:    3cc6d10cac54fd0ef44cba45038cfe5df8870372
Author: Lan Yixun (dlan) <dennis.yxun@gmail.com>
Date:   2013-07-24 11:35:39 +0800

autofs-5.0.7 add missing libtirpc lib to mount_nfs.so when TIRPC enabled

this is need when libtirpc is enabled, otherwise autofs fail to start
due to it can't find symbol clnt_dg_create (which is provided by libtirpc)


---

diff --git a/modules/Makefile b/modules/Makefile
index 8c0df18..e61e338 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -116,6 +116,6 @@ lookup_ldap.so: lookup_ldap.c dclist.o base64.o $(SASL_OBJ)
 
 mount_nfs.so: mount_nfs.c replicated.o
 	$(CC) $(SOLDFLAGS) $(CFLAGS) -o mount_nfs.so \
-		mount_nfs.c replicated.o $(AUTOFS_LIB) $(LIBNSL)
+		mount_nfs.c replicated.o $(AUTOFS_LIB) $(LIBNSL) $(TIRPCLIB)
 	$(STRIP) mount_nfs.so
